src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1150,8 +1156,27 @@ def __call__(self, *args, **kwds) -> Any:
         request.path = convert_path_to_se_path(self.path)
         request.command = self.command
         request.wait = True
+        if "FileName" in kwds:
+            kwds.update(
+                dict(
+                    FileName=os.path.basename(kwds["FileName"])
+                    if self.service.remote_file_handler.is_configured()
+                    else kwds["FileName"]
+                )
+            )
         _convert_value_to_variant(kwds, request.args)
+        file_purpose = None
+        if hasattr(self.create_instance(), "get_attr"):
+            file_purpose = self.create_instance().get_attr("FileName/filePurpose")
```

Comment:
I believe we should ideally do something like:
```
command_instance = self.create_instance()
file_purpose = command_instance.get_attr("FileName/filePurpose")
...
command_instance.delete()
```
Otherwise we would create an instance every time a command like this is called, and I don't know when/if Fluent would automatically delete or garbage collect it. 

However, I'm not sure if we have an easy way to delete created command instances from PyFluent like the `command_instance.delete()` I used as an example? Are you already aware of how to do this @hpohekar?